plugins/user/procbyuser

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+#
+# Plugin to monitor the amount of processes owned by a user
```

Comment:
Please use a perldoc header (look for `=cut` in other shell plugins for examples).

Reply:
Done.